tools/c7n_oci/c7n_oci/resources/compute.py

```diff
@@ -293,10 +293,11 @@ def process(self, resources, event):
         result = []
         for compartment_id in comp_resources.keys():
             query = self.data.get("query")
-            filter_resources = comp_resources.get(comp_id)
+            filter_resources = comp_resources.get(compartment_id)
```

Comment:
hmmm.. this makes me wonder how it passed tests before.

Reply:
we are planning to extent test case support to cover these cases as a separate PR.